terraform/aws/modules/alb/variables.tf

```diff
@@ -157,6 +157,12 @@ variable "subnets" {
   description = "the subnets to associate with the load balancer"
 }
 
+variable "security_group_ingress_cidr" {
+  type        = list(string)
+  description = "CIDR block to be configured for the ALB inbound"
+  default     = []
+}
```

Comment:
Very good - the default allows us to skip over this variable if we want to provide our own security group at the project level.